packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -1240,6 +1241,24 @@ public ReactSelectionWatcher(ReactEditText editText) {
 
     @Override
     public void onSelectionChanged(int start, int end) {
+      // Calculate cursor position
+      Layout layout = mReactEditText.getLayout();
+      if (mReactEditText.getLayout() == null) {
+            mReactEditText.getViewTreeObserver().addOnGlobalLayoutListener(new ViewTreeObserver.OnGlobalLayoutListener() {
```

Comment:
Minor thing, but I notice that the indentation of lines 1247 -> 1255 is off by two spaces

Reply:
👍🏻